lib/api/objectPutPart.js

```diff
@@ -293,17 +293,13 @@ function objectPutPart(authInfo, request, streamingV4Params, log,
                 partLocations[0].sseCipheredDataKey = cipheredDataKey;
             }
             const omVal = {
-                // Version 4 adds number-subparts for Azure data backend
-                'md-model-version': 4,
+                // back to Version 3 since number-subparts is not needed
+                'md-model-version': 3,
                 partLocations,
                 'key': partKey,
                 'last-modified': new Date().toJSON(),
                 'content-md5': hexDigest,
                 'content-length': size,
-                // NOTE: We can go back to Version 3 since we get the number of
-                // subparts from locations array
-                // NOTE: remove 'number-subparts'
-                'number-subparts': dataGetInfo.numberSubParts,
```

Comment:
please confirm this is not used anywhere in the code (searching s3 and arsenal).

Reply:
I checked, we are not